backend/coins/btc/maketx/maketx.go

```diff
@@ -168,7 +171,10 @@ func NewTxSpendAll(
 		TxOut:    []*wire.TxOut{output},
 		LockTime: 0,
 	}
-	txsort.InPlaceSort(unsignedTransaction)
+
+	secureRand := mrand.New(mrand.NewSource(secureSeed()))
```

Comment:
I'm not 100% sure, but I think that it would be better to seed the generator using a random int64 obtained from the default rand generator.  @benma wdyt?